artillery.yml

```diff
@@ -0,0 +1,13 @@
+config:
+  target: "http://127.0.0.1:3000"
+  phases:
+    - duration: 60
+      arrivalRate: 20
+
+scenarios:
+  - name: "Look at a random project"
+    flow:
+      - loop:
+        - get:
+            url: "/?{{ $randomNumber(1,10000000) }}"
+        count: 250
```

Comment:
Why are you looping a lot here?

Reply:
Looping a ton allowed me to test the maximum RPS of my server (pre-SSR) without creating a bunch of computationally expensive virtual users.